showcase/tests/integration/components/hds/dropdown/list-item/interactive-test.js

```diff
@@ -130,5 +146,24 @@ module(
         throw new Error(errorMessage);
       });
     });
+
+    // CONTEXTUAL COMPONENTS
+
+    test('it renders the contextual components', async function (assert) {
+      await render(
+        hbs`<Hds::Dropdown::ListItem::Interactive as |I|>
+              <I.Badge @text="Badge" />
+            </Hds::Dropdown::ListItem::Interactive>`
+      );
+      assert.dom('.hds-badge').hasText('Badge');
```

Comment:
question: if we decide to set the size to small implicitly, should we check for the CSS modifier?

Reply:
Yes, I think that's a good idea.